app/controllers/tournaments_controller.rb

```diff
@@ -153,6 +154,41 @@ def create_event(event)
       xtables_id: xtables_id) unless Event.find_by(xtables_id: xtables_id)
   end
 
+  def generate_two_rounds_matchups(tournament)
+    # Check how many divisions there are
+    all_players = Player.where(tournament: tournament)
+    divisions = all_players.map { |player| player.division }.uniq
```

Comment:
Do we need to do this, or can we just have a num_divisions column in the tournament table?

Reply:
Answered elsewhere, but yes this might be a useful addition at some point if I need the divisions variable elsewhere, but for now this seems like a low priority thing